oracle/src/pasta/fp_kimchi.rs

```diff
@@ -826,3 +826,10 @@ pub fn params() -> ArithmeticSpongeParams<Fp> {
         ],
     }
 }
+
+lazy_static::lazy_static! {
+    static ref PARAMS:ArithmeticSpongeParams<Fp> = params();
+}
+pub fn static_parms() -> &'static ArithmeticSpongeParams<Fp> {
```

Comment:
params*

Reply:
I was hoping to sneak the correction with other changes.